src/twisted/conch/test/test_keys.py

```diff
@@ -1025,6 +1113,28 @@ def test_toOpenSSHECDSA(self):
                 keydata.publicECDSA_openssh[:-8]) # no comment
 
 
+    def test_toOpenSSHED25519(self):
+        """
+        L{keys.Key.toString} serializes a ED25519 key in OpenSSH format.
+        """
+        key = keys.Key.fromString(keydata.privateED25519openssh)
+        self.assertEqual(key.public().toString('openssh', b'comment'),
+                         keydata.publicED25519openssh)
+
+        # We need to patch the comparison value because SecureRandom
+        # gets patched when run as a unit test.
```

Comment:
When trial is run, SecureRandom is patched by test_keys.py:217 to return '\xff' for each byte requested.

I wasn't sure how detailed to go into this explanation in the comment since it's only relevant when running trial.  I saw it that it was more important to let a normal user know that every time this function is called the output will be slightly different.